packages/netlify-cms-backend-github/src/implementation.js

```diff
@@ -221,30 +221,30 @@ export default class GitHub {
   }
 
   async entriesByFolder(collection, extension) {
-    const repoURL = `/repos/${this.useOpenAuthoring ? this.originRepo : this.repo}`;
+    const repoURL = this.useOpenAuthoring ? this.api.originRepoURL : this.api.repoURL;
     const files = await this.api.listFiles(collection.get('folder'), { repoURL });
     const filteredFiles = files.filter(file => file.name.endsWith('.' + extension));
-    return this.fetchFiles(filteredFiles, { repoURL });
+    return this.fetchFiles(filteredFiles);
```

Comment:
With the latest fix this should be `return this.fetchFiles(filteredFiles, { repoURL });`